content/20.treatments.md

```diff
@@ -110,27 +110,62 @@ Summarize the state of the antiviral approach.
+
+###### Spike (S) Neutralizing Antibody Current Evidence
+
+The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed using hybridoma technology [@doi:10.1101/2020.03.11.987958].
+The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
+The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection in vitro remains unknown, but a potential mechanism might be antibody induced destabilization of the membrane prefusion structure [@doi:10.1101/2020.03.11.987958; @doi:10.1016/j.cell.2018.12.028].
+The ability of this antibody to prevent infection at a feasible dose needs to be validated in vivo, especially since in vitro neutralization effects have been shown to not be reflective of in vivo efficacy [@doi:10.1128/JVI.01603-17].
+Only a week later, a different group reported the isolation of multiple nAbs targeting the RBD of the S glycoprotein from COVID-19 patients [@doi:10.1101/2020.03.21.990770].
+Interestingly, the patient isolated antibodies did not cross-react with RBD’s from SARS-CoV and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV and MERS-CoV was observed.
+This suggests that the RBD’s between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 is species specific.
+While specificity is desirable, it also poses the question of whether these antibodies are more susceptible to viral escape mechanisms.
+Viral escape is a common resistance mechanism to nAbs therapy due to selective pressure from neutralizing antibodies [@doi:10.2217/imt.15.33; @doi:10.1126/science.1213256].
+For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4 binding site (CD4bs) show greater neutralization breadth than monoclonal antibodies, which target only specific HIV strains [@doi:10.1016/j.molmed.2019.01.007].
+For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites prevented neutralization escape [@doi:10.1128/JVI.02002-17].
+They found that the different antibody isolates did not target the same epitopes, suggesting that using them in combination might produce a synergistic effect that prevents viral escape [@doi:10.1101/2020.03.21.990770].
+It was also demonstrated that binding affinity of the antibodies does not reflect their capability to compete with ACE2 binding.
+Furthermore, no conclusions about correlations between the severity of disease and the ability to produce neutralizing antibodies can be drawn at this point.
+Rather, higher neutralizing antibody titers were more frequently found in patients with severe disease.
+Correspondingly, another group showed that higher levels of anti-spike IgG were observed in patients that deceased from infection compared to patient that recovered [@doi:10.1172/jci.insight.123158].
```

Comment:
Correspondingly,  higher levels of anti-spike IgG were observed in patients that deceased from infection compared to patient that recovered [@doi:10.1172/jci.insight.123158].

Reply:
Sorry I have no Idea how to edit like @rando2 haha

Reply:
I put together a guide per @nilswellhausen suggestion! https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review